python-gnomeapplet is incompatible with gnome-panel-3.x

--- origsrc/hamster-applet-2.32.1/data/wscript_build	2010-11-05 20:20:33.000000000 -0500
+++ src/hamster-applet-2.32.1/data/wscript_build	2011-06-13 02:41:46.951561400 -0500
@@ -25,12 +25,6 @@ bld(features = "subst",
     dict = bld.env
    )
 
-bld(features = "subst",
-    source= "Hamster_Applet.server.in.in",
-    target= "Hamster_Applet.server.in",
-    dict = bld.env
-   )
-
 bld.add_group()
 
 # process .in files with intl_tool
@@ -47,14 +41,8 @@ bld(features = 'intltool_in',
     flags    = ['-s', '-u', '-c'])
 
 bld(features = 'intltool_in',
-    source   = """hamster-time-tracker.desktop.in
-                  hamster-applet.desktop.in""",
+    source   = 'hamster-time-tracker.desktop.in',
     install_path = '${DATADIR}/applications',
     podir    = '../po',
     flags    = ['-d', '-q', '-u', '-c'])
 
-bld(features = 'intltool_in',
-    source   = 'Hamster_Applet.server.in',
-    install_path = '${LIBDIR}/bonobo/servers',
-    podir    = '../po',
-    flags    = ['-o', '-u', '-c'])
--- origsrc/hamster-applet-2.32.1/wscript	2010-11-15 17:26:38.000000000 -0600
+++ src/hamster-applet-2.32.1/wscript	2011-06-13 02:36:18.796590500 -0500
@@ -42,7 +42,6 @@ def set_options(opt):
 
 
 def build(bld):
-    bld.install_files('${LIBDIR}/hamster-applet', 'src/hamster-applet', chmod = 0755)
     bld.install_files('${BINDIR}',
                       """src/hamster-time-tracker
                          src/hamster-cli
